assets/scss/base/_form.scss

```diff
@@ -22,8 +22,8 @@ fieldset {
 
 input {
   @include core-19();
-  background-color: $input-bg-colour;
-  border: 1px inset $input-border;
+  background-color: $white;
+  border: 2px solid #6f777b;
```

Comment:
It's a variable in govuk_elements which will be removed. I would put a colour variable in our own colours SASS file for it, but I feel I'd be confusing matters more until the refactor is done.